certbot-postfix/certbot_postfix/installer.py

```diff
@@ -44,15 +55,14 @@ class Installer(plugins_common.Installer):
     """Certbot installer plugin for Postfix.
 
     :ivar str config_dir: Postfix configuration directory to modify
-    :ivar dict proposed_changes: configuration parameters and values to
-        be written to the Postfix config when save() is called
     :ivar list save_notes: documentation for proposed changes. This is
         cleared and stored in Certbot checkpoints when save() is called
-
+    :ivar postconf: Wrapper for Postfix configuration command-line tool.
+    :ivar policy: A Policy object using which we can query
```

Comment:
Word missing here? A Policy object using what? (Or maybe "using" should be deleted.)